k8s/demo/cassandra/README.md

```diff
@@ -0,0 +1,223 @@
+-rw-rw-r-- 1 karthik karthik 2382 Nov 11 14:09 cassandra-statefulset.yaml
+```
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl apply -f cassandra-service.yaml
+service "cassandra" configured
+```
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl apply -f cassandra-statefulset.yaml
+statefulset "cassandra" created
+```
+
+Verify that all the OpenEBS persistent volumes are created, the Cassandra headless service and replicas 
+are up and running:
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl get pods
+NAME                                                             READY     STATUS    RESTARTS   AGE
+cassandra-0                                                      1/1       Running   0          4h
+cassandra-1                                                      1/1       Running   0          4h
+maya-apiserver-3416621614-g6tmq                                  1/1       Running   1          8d
+openebs-provisioner-4230626287-503dv                             1/1       Running   1          8d
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-ctrl-599202565-2kdff    1/1       Running   0          4h
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-rep-3068892500-22ccd    1/1       Running   0          4h
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-rep-3068892500-lhwdw    1/1       Running   0          4h
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-ctrl-1103031005-8vv82   1/1       Running   0          4h
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-rep-3006965094-cntx5    1/1       Running   0          4h
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-rep-3006965094-mhsjt    1/1       Running   0          4h
+```
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl get svc
+NAME                                                CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+cassandra                                           None             <none>        9042/TCP            5h
+kubernetes                                          10.96.0.1        <none>        443/TCP             14d
+maya-apiserver-service                              10.102.92.217    <none>        5656/TCP            14d
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-ctrl-svc   10.107.177.156   <none>        3260/TCP,9501/TCP   4h
+pvc-d1fee1af-bf73-11e7-9be8-000c298ff5fc-ctrl-svc   10.98.87.196     <none>        3260/TCP,9501/TCP   9d
+pvc-dfe98642-bf73-11e7-9be8-000c298ff5fc-ctrl-svc   10.101.17.255    <none>        3260/TCP,9501/TCP   9d
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-ctrl-svc   10.108.47.234    <none>        3260/TCP,9501/TCP   4h
+```
+
+Note: It may take some time for pods to start as the images need to be pulled and instantiated. This is also
+dependent on the network speed.
+
+## Verify successful Cassandra deployment
+
+The verification procedure can be carried out in a series of steps, starting from listing the functional 
+replicas, followed by creation and deletion of test data in the cassandra database.
```

Comment:
followed by creation and deletion of test data = to creating and deleting test data
cassandra = Cassandra